testing/endtoend/components/eth1/transactions.go

```diff
@@ -70,7 +70,7 @@ func (t *TransactionGenerator) Start(ctx context.Context) error {
 	}
 	f := filler.NewFiller(rnd)
 	// Broadcast Transactions every 3 blocks
-	txPeriod := 3 * time.Duration(params.BeaconConfig().SecondsPerSlot) * time.Second
+	txPeriod := time.Duration(params.BeaconConfig().SecondsPerSlot) * time.Second
```

Comment:
We send out 100 transactions every slot so that it allows the builder to always be active. Otherwise the beacon node will reject a payload with 0 txs from the builder.

Reply:
The comment says
> // Broadcast Transactions every 3 blocks